wolfcrypt/src/port/caam/caam_driver.c

```diff
@@ -29,11 +34,11 @@
     #include <sys/iofunc.h>
     #include <sys/neutrino.h>
 
-    #include "caam_qnx.h"
+    #include <wolfssl/wolfcrypt/port/caam/caam_qnx.h>
```

Comment:
These header files are not installed on the system, not located in a /usr/local/include/ location.

Reply:
There is already an include path to wolfssl-root. The "camm_qnx.h" requires setting an include path to `wolfssl/wolfcrypt/port/caam`. Is this change really going to cause a problem? I assume the caam build is keeping the header file folder structure still, right?